README.md

```diff
@@ -19,9 +19,9 @@ Hopefully, it will get better soon.
     * It should look like:
     ```
     [github]
-       token = "my_token"
+       token = my_token
     ```
-3. Store token under `GITHUB_TOKEN` in the environment.
+   Alternatively, you can store the token under `GITHUB_TOKEN` in the environment.
```

Comment:
I am not sure if the env variable is not needed. it was not always working only with the config

Reply:
see above comment